api/origin_trials_api.py

```diff
@@ -47,6 +48,48 @@ def get_chromium_file(url: str) -> str:
     return  b64decode(conn.read()).decode('utf-8')
 
 
+def get_chromium_files_for_validation() -> dict:
+  """Get all chromium file contents stored in a dictionary"""
+  chromium_files = {}  # Chromium source file contents.
+  with concurrent.futures.ThreadPoolExecutor(max_workers=5) as executor:
```

Comment:
Do we need 5 workers? CHROMIUM_SRC_FILES only has 4 files so far

Reply:
It seems fine to drop it to 4 to match 🙂